uptane-standard.md

```diff
@@ -289,20 +289,309 @@ The Timestamp role SHALL produce and sign metadata indicating whether there are
+
+The following example specifies all of the REQUIRED as well as all of the RECOMMENDED attributes for the body of targets metadata.
+```
+Number of Targets  = Natural ; Allowed to have no targets at all.
+TargetsMetadata    = NumberOfTargets Targets [TargetsDelegations] ; https://tools.ietf.org/html/rfc6025#section-2.4.2
+
+; Adjust length to your needs.
+Targets           = 1*128(TargetAndCustom)
+TargetAndCustom   = Target [Custom]
+NumberOfHashes    = Length
+Target            = URL Filename Length NumberOfHashes Hashes
+
+; The release counter is used to prevent rollback attacks on images when
+; only the director repository is compromised.
+; Every ECU should check that the release counter of its latest image is
+; greater than or equal to the release counter of its previous image.
+ReleaseCounter = Natural
+
+; The hardware identifier is used to prevent the director repository,
+; when it is compromised, from choosing images for an ECU that were not
+; meant for it.
+; Every ECU should check that the hardware ID of its latest image matches
+; its hardware ID.
+; An OEM MAY define other types of information to further restrict the
+; choices that can be made by a compromised director repository.
+HardwareIdentifier = Identifier
+
+; The ECU identifier specifies information, e.g., serial numbers, that the
+; director uses to point ECUs as to which images they should install.
+; Every ECU should check that the ECU ID of its latest image matches its
+; own ECU ID.
+EcuIdentifier = Identifier
+
+; This attribute MAY be used by the director to encrypt images per ECU.
+EncryptedTarget = Target
+
+; This attribute is used to specify additional information, such as which
+; images should be installed by which ECUs, and metadata about encrypted
+; images.
+; NOTE: The first 2 attributes are specified by both the image and
+; director repositories.
+; NOTE: The remaining attributes are specified only by the director
+; repository.
+Custom = [ReleaseCounter] [HardwareIdentifier] [EcuIdentifier] [EncryptedTarget] [EncryptedSymmetricKey]
+
+; This is the symmetric key encrypted using the asymmetric ECU key.
+EncryptedSymmetricKeyValue = BinaryData
+
+; This attribute MAY be used if ECU keys are asymmetric, and a per-image
+; symmetric encryption key is desired for faster decryption of images.
+; In that case, the director would use the asymmetric ECU key to encrypt
+; this symmetric key.
+EncryptedSymmetricKey = EncryptedSymmetricKeyType EncryptedSymmetricKeyValue
+
+EncryptedSymmetricKeyType = "aes128" / "aes192" / "aes256" / ...
+
+; The public keys of all delegatees.
+NumberOfKeys = Length
+
+; The role name, filename, public keys, and threshold of a delegatee.
+NumberOfDelegations = Length
+
+; A list of paths to roles, listed in order of priority.
+Delegations = PrioritizedPathsToRoles
+
+TargetsDelegations  = NumberOfKeys PublicKeys NumberOfDelegations Delegations
+
+; Adjust length to your needs.
+PrioritizedPathsToRoles = 1*8(PathsToRoles)
```

Comment:
Why don't we follow this `1*N(Object)` convention almost everywhere else? It's hard to read otherwise, I think...